tests/format/less/case/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -321,3 +321,131 @@ TABLE {
 
 ================================================================================
 `;
+
+exports[`variable.less format 1`] = `
+====================================options=====================================
+parsers: ["less"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+
+
+// do not change case of property name if inside a variable declaration
+@var: {
+  notVar: 0;
+  @notVarNested: {
+    notVar: 1;
+    notVar2: 2;
+  };
+};
+
+@var    :    {
+  shouldNotChangeCase: 5;
+};
+
+@var:    {
+  shouldNotChangeCase: 5;
+};
+
+@var    :{
+  shouldNotChangeCase: 5;
+};
+
+@var /* comment */ : /* comment */ {
+  shouldNotChangeCase: 5;
+};
+
+@var: /* comment */ {
+  shouldNotChangeCase: 5;
+};
+
+@var /* comment */ :{
+  shouldNotChangeCase: 5;
+};
+
+@var // comment
+: // comment
+{
+  shouldNotChangeCase: 5;
+};
+
+@var: // comment
+{
+  shouldNotChangeCase: 5;
+};
+
+@var // comment
+:{
+  shouldNotChangeCase: 5;
+};
+
+@not-var {
+  shouldNotChangeCase: @var[@notVarNested][notVar];
+  shouldNotChangeCase: @var[    @notVarNested][notVar];
+  shouldNotChangeCase: @var[@notVarNested    ][notVar];
+  shouldNotChangeCase: @var[@notVarNested][    notVar];
+  shouldNotChangeCase: @var[@notVarNested][notVar    ];
+  shouldNotChangeCase: @var[notVar];
+}
+
+=====================================output=====================================
+// do not change case of property name if inside a variable declaration
+@var: {
+  notVar: 0;
+  @notVarNested: {
+    notVar: 1;
+    notVar2: 2;
+  };
+};
+
+@var: {
+  shouldNotChangeCase: 5;
+};
+
+@var: {
+  shouldNotChangeCase: 5;
+};
+
+@var: {
+  shouldNotChangeCase: 5;
+};
+
+@var: /* comment */ {
+  shouldnotchangecase: 5;
+};
+
+@var: /* comment */ {
+  shouldNotChangeCase: 5;
+};
+
+@var: {
+  shouldnotchangecase: 5;
+};
+
+@var // comment
+: // comment
+{
+  shouldnotchangecase: 5;
+}
+
+@var: // comment
+{
+  shouldNotChangeCase: 5;
+};
+
+@var // comment
+: {
+  shouldnotchangecase: 5;
+}
+
+@not-var {
+  shouldnotchangecase: @var[ @notVarNested][notVar];
+  shouldnotchangecase: @var[ @notVarNested][notVar];
+  shouldnotchangecase: @var[ @notVarNested ][notVar];
+  shouldnotchangecase: @var[ @notVarNested][ notVar];
+  shouldnotchangecase: @var[ @notVarNested][notVar ];
```

Comment:
We are fine with these spaces, right?

Reply:
I would expect no trailing space, especially when space is not present in the original. Can it be fixed easily?

Reply:
I didn't notice the space is added.